webmails/rainloop/config/nginx-rainloop.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    root /var/www/rainloop;
+    
+    # /dev/stdout (Default), <path>, off
+    access_log off;
+
+    # /dev/stderr (Default), <path>, debug, info, notice, warn, error, crit, alert, emerg
+    error_log /dev/stderr error;
+
+    index index.php;
+
+    location / {
+        try_files $uri $uri/ /index.php?$query_string;
```

Comment:
Received feedback to remove `$uri/` which is required when installing `nginx` on `alpine:3.14`.

Related issue: https://stackoverflow.com/questions/19285355/nginx-403-error-directory-index-of-folder-is-forbidden